### DISCUSSION TITLE
挂载后局域网不用密码登录

### DISCUSSION THREAD

**Discussion by zjw345** [AlistGo/alist#531] [Ideas]
挂载后局域网不用密码登录就可以下载网盘文件，不安全。建议访问ip：5244时候需要密码。

**Comment by xhofe** [comment]
不知道你在说什么

**Reply by zjw345** [reply]
我是群辉docker安装的，本想挂载为webdav，挂载后发现，ip:5244网页端不用密码，就能直接查看和下载我网盘里的内容了。能不层访问这个网页前端都设置密码？

**Reply by xhofe** [reply]
首先这个程序的初始定位就是公开的列表分享，加密只是附加的功能。
其次你应该先看文档：https://alist-doc.nn.ci/docs/meta/path

**Reply by zjw345** [reply]
原来有这个帮助文档，非常感谢。

**Comment by simpleandstupid** [comment]
如果不想被局域网内设备访问，监听127.0.0.1不会有任何安全问题，修改[config.json](https://alist-doc.nn.ci/docs/setting/config)
"address": "127.0.0.1",

**Reply by zjw345** [reply]
我是docker安装的，config.json在哪个位置？

**Reply by i-sync** [reply]
> docker run -d --restart=always -v /etc/alist:/opt/alist/data -p 5244:5244 --name="alist" xhofe/alist:v2

/etc/alist 目录

**Reply by isenwen** [reply]
> 如果不想被局域网内设备访问，监听127.0.0.1不会有任何安全问题，修改[config.json](https://alist-doc.nn.ci/docs/setting/config) "address": "127.0.0.1",

如果我是用的外网服务器搭建的呢？有办法不让其他人访问吗

**Reply by anwen-anyi** [reply]
> > 如果不想被局域网内设备访问，监听127.0.0.1不会有任何安全问题，修改[config.json](https://alist-doc.nn.ci/docs/setting/config) "address": "127.0.0.1",
> 
> 如果我是用的外网服务器搭建的呢？有办法不让其他人访问吗

**建议直接去元信息把首页加密并且隐藏同时去后端开启检查父目录密码**.......
